### ISSUE TITLE
Earth Dragon Concept

### ISSUE THREAD

**Issue by CMDRRoach** [AlexModGuy/Ice-and-Fire-Suggestions#1340]
since i dont think a just ad that would be any help i thought to lay out a dietailed concept on how and Earth Dragon could work in the game

Concept:
the Earth Dragon would of course look fairly similar to the other dragons with one slight difference he has thicker scales and even if hes the same size per tier as the other dragons he has a very bulky apperance reason for that is his territorry while other dragons besides T4-T5 live on the ground the Earth Dragon mostly lives underground or within mountains slighly above see level besides cave dwelling animals and the occasional unlucky adventurer his died mostly consits out the minerals within his caves this died affects his apperance and his attack style for example his apperance is not just for show while he has less base hp then other dragon spiecies hes got much more armor then the others so much that hes only able to leap and glide (at first) while for his breath an meteor atacks he uses a dust could made from bone meal coal and sulfur that lingers and blinds his fows and not only that he can ignite the lingering dust clouds causing them to explode for his meteor he uses the same dust mixed with his seliva to create a ball of sticky tar that slows and weakens within the aos area while a direct hit causes the target to get temporarly petrefied and suffer from block suffucation dmg unless his released by a fellow adventurers pickaxe for his weaknesses his breath can also work against him if his struck at the head with a flame enchatnet bow the moment he prepares his breath attack can cause the dragon to explode however while it can deal a lot of dmg this wont kill him in stead his heavy scales blow off reducing his armor but enableing the dragon to fly

Tamed behavior and benefinds
the cant fly with fully developed scales how ever he can tear throught the ground  eat ores in the process that can be mined from his back with a pick axe when his mineral scales are mines the dragon becomes light enough to fly
Breath attack can be used in short bursed to fertelize fields or in or can be exploded when R is hold for more then 5 seconds
color variants: andesite grey, granit red, diorite white, deep slate black and moss green 

to summarize
aperrance bulkyer then other dragons
lives in mountain biomes t3 within mountains above sea level t4-t5 at regular dragon cave depth
breaths blinding dust that can explode or spits tar that can slow, weaken and petrefie
can dig through terrain and eat ores that can be mined later form his back
can fertilize your crops

i thought about this concept on a bus ride to work after playing on a friends server that had ice and fire installed so i though why not give it a shot
